### ISSUE TITLE
Multisig Creation Issue (v.0.9.3)

### ISSUE THREAD

**Issue by yurieasy** [ethereum/mist#3429]
- [+] I've asked for help in the [Mist Gitter](http://gitter.im/ethereum/mist) before filing this issue.

Hello colleagues, 

I have an issue creating multisig in Ethereum Wallet (v.0.9.3).

I’ve passed basic user interface process for creation the multisig (3/4 owners with daily limit 10)
And the creation did not happen.
Creation Icon looks spinning ( https://cl.ly/201v3s1S0R2B ).
Transaction is not sent to blockchain (I looked both in geth console and on etherscan).

Do you have any ideas about this issue?

```
Version: `0.9.3`
OS & Version: osx
Node version: `geth 1.7.2` 
Number of blocks synchronized: 4,691,905
```

**Comment by evertonfraga** [comment]
Yes, and there's a potential fix for that issue, coming in the next version.

As a workaround, you could download and run Geth (https://geth.ethereum.org) separately with the command:
`geth --syncmode light`

Then, only when it connects properly, you can start Mist again.

**Comment by evertonfraga** [comment]
Fixed in 0.10.0.
